src/test/java/com/courtcanva/ccfranchise/services/OrderServiceTest.java

```diff
@@ -1,37 +1,59 @@
 package com.courtcanva.ccfranchise.services;
 
-import static com.courtcanva.ccfranchise.utils.OrderTestHelper.orders;
-import static org.junit.jupiter.api.Assertions.assertEquals;
-import static org.junit.jupiter.api.Assertions.assertTrue;
-import static org.mockito.Mockito.when;
-
 import com.courtcanva.ccfranchise.constants.OrderStatus;
 import com.courtcanva.ccfranchise.dtos.orders.OrderGetDto;
+import com.courtcanva.ccfranchise.mappers.OrderMapper;
 import com.courtcanva.ccfranchise.mappers.OrderMapperImpl;
 import com.courtcanva.ccfranchise.repositories.OrderRepository;
+import com.courtcanva.ccfranchise.utils.FranchiseeTestHelper;
+import com.courtcanva.ccfranchise.utils.OrderTestHelper;
+import static com.courtcanva.ccfranchise.utils.OrderTestHelper.orders;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+import static org.mockito.Mockito.when;
 import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.extension.ExtendWith;
 import org.mockito.Mock;
 import org.mockito.junit.jupiter.MockitoExtension;
+import org.springframework.data.domain.PageRequest;
+
+import static org.junit.jupiter.api.Assertions.*;
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.when;
 import java.math.BigDecimal;
 import java.util.ArrayList;
 import java.util.List;
 
 @ExtendWith(MockitoExtension.class)
 class OrderServiceTest {
 
-    private OrderService orderService;
-
     @Mock
     private OrderRepository orderRepository;
 
+    private OrderService orderService;
+
+    private OrderMapper orderMapper;
```

Comment:
this `orderMapper` is only used in `setUp()`, so it's not good to declare here, declare in `setUp()` is better.

Reply:
seems you didn't modify.